### PR TITLE
Week 1 Homework and Roster update

### PR DESCRIPTION
**Pull request by keviny22**

### CODE REVIEW

**Comment by keviny22** [comment]
Added week 2

**Comment by keviny22** [comment]
@reneedv I wasn't able to do another pull request for week 2 since the week 1 PR wasn't closed. Is that normal or am I doing something wrong?

**Comment by bethanyr** [comment]
Kevin - yes- until the pull request has been reviewed and closed, any changes to your branch will show up in this pull request.

File: week2/homework/questions.txt [hunk 1]
```diff
@@ -3,11 +3,33 @@ Containers, Blocks, and Iterators
...
+I use a hash when i would need to lookup a value based on its key, or vice
+versa.
 
 3. What is a module? Enumerable is a built in Ruby module, what is it?
+a way of grouping together methods, classes, and constants
+an enumerable is a standard mixin, which combines a bunch of methods, i like
+the map enumerable which makes running blocks of code much cleaner
 
 4. Can you inherit more than one thing in Ruby? How could you get around this problem?
+use mixins
```
Comment (bethanyr) [inline week2/homework/questions.txt:26]: multiple inheritance is not allowed in Ruby - you are correct you would use mix-ins or multiple modules.

**Comment by bethanyr** [comment]
Week 1 & Week 2 homework look good.

**Pull request closed without merging**